app/assets/javascripts/client/inventories/InventoryReportSidebarCtrl.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('InventoryReportSidebarCtrl', InventoryReportSidebarCtrl);
+
+  InventoryReportSidebarCtrl.$inject = [
+    '$scope',
+    '$modal',
+    'inventory',
+    '$stateParams',
+    'SessionService',
+    '$state'
+  ];
+
+  function InventoryReportSidebarCtrl($scope, $modal, inventory, $stateParams, SessionService, $state) {
+    var vm = this;
+    $scope.inventory = inventory;
+    vm.inventory = $scope.inventory;
+    vm.shared = $stateParams.shared || false;
+    var user = SessionService.getCurrentUser();
+    vm.hideAnalysisAccess = inventory.analysis_count == 0 && user.role == 'participant';
```

Comment:
Two things here:

 - We don't want to be checking the user's role.  Recall that this data can be mutated elsewhere, and the information about the user's role with respect to this inventory is not stored here.  You would want to use `inventory.is_facilitator`.
 - Use the `===` operator instead of `==`, as `==` will try to convert the types before comparing equality.  The only reliable way to compare is with `===`.